front page

ric-e opened the pull request:
This updates the layout.svelte and the page.svelte for the front page and the permanent display of the website. 

in layout.svelte, it added a NavBar that contains a theme switch (permanent theme tbd), the title of the website, a profile button, and a Menu dropdown

in page.svelte, there has been no permanent updates. a temporary placeholder shows an animated loading sign

there will be a bottom bar in the future if need be

ric-e committed:
need jack's feedback

ric-e committed:
edited the profile drowdown

ric-e commented:
Need Jack's feedback and help on:

- keeping theme after page refresh
- dropdowns close if another is selected 
- avoiding text wrapping in dropdowns
- Integrating login system
- images and graphs (putting pictures in our title page)

ric-e committed:
used a temporary default profile pic for web development before the user uploads their own, used tinyurl.com to generate a shortened image addresss

ric-e committed:
not working well. need to change font colouring and fix the animation not playing

ric-e committed:
fixed problem with open <ul> tag

ric-e committed:
fixing things

ric-e committed:
does not work the buttons are stupid

ric-e commented:
@jackkoskie need some help, the buttons only click on the borders for the dropdown ones on the right of navbar, and the theme toggler has broken altogether (i assume with the same issue). I recopied the code from DaisyUI and looked at my old commits when the buttons still work but i can't figure out whats wrong

jackkoskie committed:
remove styling that translates

jackkoskie commented:
@ric-e The translate styling that you had on the main page was moving the body ontop of the navbar meaning you were no longer clicking the navbar but rather the body. I removed the conflicting code

ric-e committed:
Update +layout.svelte

ric-e committed:
fixed margins on navbar

ric-e committed:
fixed link something because default profiel pic was crying

jackkoskie committed:
make theme switcher maintain state

jackkoskie committed:
Merge branch '20-front-page' of https://github.com/jackkoskie/chill-pill into 20-front-page

jackkoskie committed:
fix theme saving

ric-e committed:
fixed theme

jackkoskie committed:
add link from logo to home

ric-e committed:
yes

ric-e committed:
Merge branch '20-front-page' of https://github.com/jackkoskie/chill-pill into 20-front-page

ric-e committed:
sad#

ric-e committed:
added a non-sticky footer

ric-e committed:
.

ric-e committed:
commited dush's color scheme (testing atm)

ric-e commented:
Note for Jan 24:

- Added color scheme that dush made (not all colors are changed from default because his logo only has a few colors
- added a bottom footer (no working links yet)
- working on fixing dropdowns (not commited yet)

ric-e committed:
adjusted z-index of the navbar and footer to be maximum possible (9999). justified and fixed the bullet points for the center text (not done the styling) and centered + justified it

ric-e committed:
font changed

jackkoskie committed:
fix theme toggle

ric-e committed:
fixed logos in footer, linked to new tab

ric-e committed:
navbar logo

ric-e committed:
Create chill_pill_corner.png

ric-e committed:
Update tailwind.config.ts

ric-e commented:
Implement internationalization @jackkoskie next class remind me

ric-e committed:
changed theme icons to inconify (getting rid of svgs)

jackkoskie reviewed (approved):
lgtm
